R/compareScenConf.R

```diff
@@ -13,15 +13,15 @@
 #'  \dontrun{
 #'     compareScenConf(fileList = c("scenario_config1.csv", "scenario_config2.csv"),
 #'     configfile = "default.cfg",
-#'     renamedColumns = c("old1" = "new1", "old2" = "new2"),
-#'     renamedRows = c("old3" = "new3", "old3" = "new4"))
+#'     renamedCols = c("old1" = "new1", "old2" = "new2"),
+#'     renamedRows = c("old3" = "new3", "old4" = "new4"))
```

Comment:
Are you sure `renamedColumns` as a function name is not used anywhere? Why change the API here?

Reply:
Well, as this function is only some days old, nobody knows about it and `reposearch` founds nothing, so I do not suspect any problems. Why? Just less to type…